src/user/LoginResolver.ts

```diff
@@ -53,7 +53,11 @@ export class LoginResolver {
     const cache = this.hostnameSignedMessageHashCache;
     if (cache[hostname]) return cache[hostname];
 
-    if (!this.hostnameWhitelist.has(hostname)) return null;
+    if (
+      !this.hostnameWhitelist.has(hostname) ||
```

Comment:
```suggestion
      !this.hostnameWhitelist.has(hostname) &&
```
Current implementation returns null if hostname is not in the whitelist

Reply:
Thanks! No more midnight coding